keras_nlp/utils/text_generation.py

```diff
@@ -17,6 +17,53 @@
 import tensorflow as tf
 
 
+def validate_prompt(prompt):
+    """
```

Comment:
Hmm, with the whole docstring for a helper it's hard to tell it's just a helper function. In general I don't think you would need the whole args/return structure for something small like this.

Just `Helper function to validate input to text_generation utils.`